src/Microsoft.VisualStudio.Web.CodeGeneration.Templating/TypeUtilities.cs

```diff
@@ -22,7 +21,6 @@ private static bool IsNonIntegerPrimitive(Type type)
                    || (type == typeof(double))
                    || (type == typeof(float))
                    || (type == typeof(Guid))
-                   || (type == typeof(string))
                    || (type == typeof(TimeSpan))
```

Comment:
It's not clear to me why string would be removed from this check, I'm missing some context.